{"name": "test_sign_up_invalid_phone_num3", "status": "passed", "description": "the test is verify, that user can't sign up with invalid phone number", "start": 1680535735484, "stop": 1680535736298, "uuid": "d8ca0f1a-8532-4982-8852-8fd22ac3e5f9", "historyId": "80051cee47059ac718aa3cfe221dd2de", "testCaseId": "98cdbe3a2daee48437d0250a2dccc868", "fullName": "tests.chromeTests.testSignUpModal.TestSignUpModal#test_sign_up_invalid_phone_num3", "labels": [{"name": "parentSuite", "value": "tests.chromeTests"}, {"name": "suite", "value": "testSignUpModal"}, {"name": "subSuite", "value": "TestSignUpModal"}, {"name": "host", "value": "krot"}, {"name": "thread", "value": "19772-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.chromeTests.testSignUpModal"}]}